frameworks/helloworld/src/main/dist/svc.yml

```diff
@@ -33,3 +40,8 @@ pods:
             size: {{WORLD_DISK}}
         env:
           SLEEP_DURATION: {{SLEEP_DURATION}}
+        readiness-check:
+          cmd: "cat world-container-path/output"
```

Comment:
Nit, but as a demonstration of readiness checks, should this maybe have a slightly different success criterion than the health check? It seems like this would succeed with return code 0 in all cases where the invocation of `stat` would.